src/www/widgets/widgets/ipsec.widget.php

```diff
@@ -68,7 +68,19 @@
         }
     }
 }
-
+// Initialize variable aggregated_data and loop through the ipsec_leases array to fetch the data for user, address and online status. Used later for div ipsec-mobile.
+$aggregated_data = array();
+foreach ($ipsec_leases as $lease) {
+    $user = $lease['user'];
+    $address = $lease['address'];
+    $online = $lease['online'];
+    // For each unique user, initialize an empty array
+    if (!isset($aggregated_data[$user])) {
```

Comment:
It's shorter to use the `$lease['user']` without storing it first (e.g. `if (!isset($aggregated_data[$lease['user']])) {`), although for `$user` I can understand it if one would find it more readable, the address and online fields are certainly less useful to store first (just adding `$lease` to `$aggregated_data[$user]` might also be an option, the extra user field is not used anyway.